src/neigh.c

```diff
@@ -639,6 +640,15 @@ int neigh_output(int af, union inet_addr *nexhop,
             neigh_entry_state_trans(neighbour, 0);
         }
 
+        /*
+         * Sync to other cores though it has not been resolved. And the reason
+         * is that neighbour solicitation is sent from core A where the related
+         * neighbour entry is created. However the related neighbour
+         * advertisement is received on core B. So if not synced, we have no
+         * chance to resolve it.
+         */
+        neigh_sync_core(neighbour, 1, NEIGH_ENTRY);
+
```

Comment:
It's OK for core A send ns and core B received na. 'ndisc_recv_ns' will create entry in core B and do 'neigh_sync_core' . Did you have some problem with it?

Reply:
It works fine and I will raise a new pull request.